code/modules/mining/machine_processing.dm

```diff
@@ -5,6 +5,36 @@
 /obj/machinery/mineral
 	var/input_dir = NORTH
 	var/output_dir = SOUTH
+	var/turf/input_turf = null
+	processing_flags = START_PROCESSING_ON_INIT | FAST_PROCESS_SPEED

```

Comment:
This is actually incorrect for you want it to do. You dont want it like this as it will actually be on fast process from loadup. You want the signal to start the processing and to then ignore all other calls until the process() tells it to stop and start listening again. All you have done at this moment in time is have it on fast process waiting for minerals.

Reply:
I'm don't think I understand what you're saying here. These are the flags for the parent mineral machines. I've overridden the ORMs first flag with START_PROCESSING_MANUALLY so it won't start up when the round first starts up. I do wan't all the mineral machines to be on fast processing as that's what they were on before.